File: algorithm/python/BOJ2750.py
# problem : https://www.acmicpc.net/problem/2750

import sys

N = int(sys.stdin.readline().rstrip())
arr = list()

for i in range(0, N):
    num = int(sys.stdin.readline().rstrip())
    arr.append(num)

arr.sort()

for num in arr:
    print(num)

File: algorithm/typescript/BOJ2750.ts
// problem : https://www.acmicpc.net/problem/2750

import fs = require("fs");

const input: number[] = fs.readFileSync("/dev/stdin").toString().split("\n").map(Number);
const arr: number[] = [];
const N: number = input[0];

for (let i = 1; i <= N; i++) {
  let num: number = input[i];
  arr.push(num);
}

arr.sort((a, b) => {
  return a - b;
});

console.log(arr.join("\n"));
